docs/_docs/testnet/changelog.md

```diff
@@ -2,8 +2,15 @@
 ---
 # Obscuro Testnet Change Log
 
-# Oct 2023-10-06 (v0.18.0)
+# Oct 2023-10-11 (v0.18.0)
```

Comment:
Weird that these changes are already on main?

Reply:
I might have botched it when merging